src/redux/configureStore.js

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux';
+import rootReducer from './reducers';
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [ ] Good job on this implementation, but the project requires you to use `redux-toolkit` which is a modern and simpler way of implementing redux. Please start by importing `configureStore` from `redux-toolkit` and then make necessary adjustments to other parts.